rosa_getting_started/rosa-getting-started-workflow.adoc

```diff
@@ -1,22 +1,29 @@
+[id="rosa-understanding-the-deployment-workflow"]
+= Understanding the ROSA deployment workflow
 include::modules/attributes-openshift-dedicated.adoc[]
-
-[id="rosa-getting-started-workflow"]
-= Getting started workflow
-:context: rosa-getting-started-workflow
+:context: rosa-understanding-the-deployment-workflow
 
 toc::[]
 
-Follow this workflow to set up and access {product-title} (ROSA) clusters:
+Before you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you must complete the AWS prerequisites, verify that the required AWS service quotas are available, and set up your environment.
```

Comment:
Consider an unordered list.

Reply:
On reflection, given that there are only three items and that there are ordered steps in the next section that give the full workflow in detail, I will leave these as is. It's not a complete list in itself and if we change this to an unordered list, it might be necessary to include all of the workflow steps. This would duplicate the subsequent section.